config/routes.rb

```diff
@@ -0,0 +1,19 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+   root 'welcome#index'
+   resources :works
+   resources :users, except: [:new, :edit, :update, :delete]
```

Comment:
With a list like this, you might want to use the "only" syntax to get index and show